.github/workflows/pylint.yml

```diff
@@ -0,0 +1,24 @@
+name: Pylint
```

Comment:
looks good. But do you know how to add this as its own PR, so it isn't connected to the login PR?
See: https://stackoverflow.com/questions/8450036/how-to-open-multiple-pull-requests-on-github

Reply:
noted, i will put the fixes in a new PR to make sure i understand

Reply:
oh, just realized did mistake. i will push the whole changes in new pull request. sorry :(